apmpackage/apm/data_stream/traces/elasticsearch/ingest_pipeline/default.yml

```diff
@@ -15,6 +15,16 @@ processors:
       name: client_geoip
   - pipeline:
       name: event_duration
+  # event.success_count logic
+  - set:
+      if: ctx.event?.outcome == 'success'
+      field: event.success_count
+      value: 1
+  - set:
+      if: ctx.event?.outcome == 'failure'
+      field: event.success_count
+      value: 0
```

Comment:
I'd also add `"ignore_failure": true`.

Reply:
Added. But it isn't obvious under what circumstances this will fail, given the null event check. That is why I didn't add `ignore_failure` in the first place. Do you have any hints?

Reply:
agreed, no obvious circumstance, but not having it did bite us in the past, so we basically have it on all processors.